### DISCUSSION TITLE
Password for tenant databases?

### DISCUSSION THREAD

**Discussion by KatiaSishost** [spatie/laravel-multitenancy#293] [Q&A] [answered]
when i ussing the multi-database config... it's possible to save the user and password from the new users in some database or other?

**Comment by masterix21** [comment, marked as answer]
Hi @KatiaSishost, sure. 

- add the needed fields in the landlord's tenant table
- create a custom database switch task that uses your new fields
- replace in the `multitenancy.conf` our database switch task with yours

Happy coding.
